pkg/helpers/helpers.go

```diff
@@ -372,31 +372,25 @@ func ApplyResources(clientHolder *ClientHolder, recorder events.Recorder,
 
 		switch required := obj.(type) {
 		case *corev1.ServiceAccount:
-			_, _, err := resourceapply.ApplyServiceAccount(clientHolder.KubeClient.CoreV1(), recorder, required)
+			_, _, err := resourceapply.ApplyServiceAccount(context.TODO(), clientHolder.KubeClient.CoreV1(), recorder, required)
 			errs = append(errs, err)
 		case *corev1.Secret:
-			_, _, err := resourceapply.ApplySecret(clientHolder.KubeClient.CoreV1(), recorder, required)
+			_, _, err := resourceapply.ApplySecret(context.TODO(), clientHolder.KubeClient.CoreV1(), recorder, required)
 			errs = append(errs, err)
 		case *corev1.Namespace:
-			_, _, err := resourceapply.ApplyNamespace(clientHolder.KubeClient.CoreV1(), recorder, required)
+			_, _, err := resourceapply.ApplyNamespace(context.TODO(), clientHolder.KubeClient.CoreV1(), recorder, required)
 			errs = append(errs, err)
 		case *appsv1.Deployment:
 			errs = append(errs, applyDeployment(clientHolder, recorder, required))
 		case *rbacv1.ClusterRole:
-			_, _, err := resourceapply.ApplyClusterRole(clientHolder.KubeClient.RbacV1(), recorder, required)
+			_, _, err := resourceapply.ApplyClusterRole(context.TODO(), clientHolder.KubeClient.RbacV1(), recorder, required)
 			errs = append(errs, err)
 		case *rbacv1.ClusterRoleBinding:
-			_, _, err := resourceapply.ApplyClusterRoleBinding(clientHolder.KubeClient.RbacV1(), recorder, required)
-			errs = append(errs, err)
-		case *crdv1beta1.CustomResourceDefinition:
```

Comment:
we need this for ocp 311, as a temporary way, you can copy the old code to here

Reply:
changed, please reiview it again